bot/exts/moderation/modlog.py

```diff
@@ -522,14 +522,27 @@ async def log_cached_deleted_message(self, message: discord.Message) -> None:
                 f"**Message ID:** `{message.id}`\n"
                 f"**Sent at:** {format_dt(message.created_at)}\n"
                 f"[Jump to message]({message.jump_url})\n"
-                "\n"
             )
 
+        # If the message is a reply, add the reference to the response
+        if message.reference is not None and message.reference.resolved is not None:
+            resolved_message = message.reference.resolved
+
+            if isinstance(resolved_message, discord.Message):
+                jump_url = resolved_message.jump_url
+                author = resolved_message.author.mention
+
+                reference_line = (
+                    f"**In reply to:** {author} [Jump to message]({jump_url})\n"
```

Comment:
As we have another "Jump to message" later, I'd like to avoid any confusion if someone clicks it without paying attention. Maybe make it say "Jump to referenced message"

Reply:
Alright! ill change it